src/System.CommandLine/Help/HelpOption.cs

```diff
@@ -1,24 +1,19 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System.CommandLine.Invocation;
-using System.CommandLine.IO;
-using System.Threading;
-using System.Threading.Tasks;
-
 namespace System.CommandLine.Help
 {
-    internal class HelpOption : Option<bool>
+    public sealed class HelpOption : Option<bool>
     {
-        internal HelpOption(string name, string[] aliases)
+        public HelpOption(string name, string[] aliases)
             : base(name, aliases, new Argument<bool>(name) { Arity = ArgumentArity.Zero })
         {
             AppliesToSelfAndChildren = true;
             Description = LocalizationResources.HelpOptionDescription();
-            Action = new HelpOptionAction();
+            Action = new HelpAction();
```

Comment:
Should we create this lazily? I think the ability to do so is important so we don't do up-front configuration for every action including the ones that don't end up being used for a particular invocation.

Reply:
Very good point. I can make it possible, but it's going to require making the `CliAction` property virtual. Which should be fine?